streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -1348,14 +1346,7 @@ private String decorateTopic(final String topic) {
     }
 
     void initializeSubscription() {
```

Comment:
Do we need this empty method really?

Reply:
Oh I just forgot to remove the method, thanks